src/Sulu/Bundle/RouteBundle/Tests/Unit/Generator/SymfonyExpressionTokenProviderTest.php

```diff
@@ -21,31 +21,36 @@ class SymfonyExpressionTokenProviderTest extends TestCase
     public function testResolve()
     {
         $translator = $this->prophesize(TranslatorInterface::class);
+        $translator->getLocale()->willReturn('de');
+        $translator->setLocale('de')->shouldBeCalled();
```

Comment:
@adriansieber theoretically the translator should also be called with `en` because the entity has the function `getLocale` and return `en` - can you check why this is not happening?